packages/hash/frontend/src/components/GlideGlid/use-grid-tooltip/grid-tooltip-manager.ts

```diff
@@ -0,0 +1,119 @@
+import type { DrawArgs } from "@glideapps/glide-data-grid/dist/ts/data-grid/cells/cell-types";
+import { CustomGridIcon } from "../custom-grid-icons";
+import { TooltipCell } from "./types";
+
+export class GridTooltipManager {
+  // tooltip size
+  private size = 20;
+
+  // gap between tooltips
+  private gap = 10;
```

Comment:
Doesn't block merging this PR, but if you think that these variable names alone are insufficient to understand their purpose, you could call them `tooltipSize` and `gapBetweenTooltips` and lose the comments – and the variables would be self-describing everywhere they appeared

Reply:
I agree with this, makes sense. Gonna do this on follow-up pr to prevent conflicts.